pkgs/state-transition/src/transition.zig

```diff
@@ -241,6 +244,20 @@ fn process_attestations(allocator: Allocator, state: *types.BeamState, attestati
             _ = justifications.remove(vote.target.root);
             logger.debug("\n\n\n-----------------HURRAY JUSTIFICATION ------------\n{any}\n--------------\n---------------\n-------------------------\n\n\n", .{state.latest_justified});
 
+            // Emit new justification event via SSE (only in non-ZKVM contexts)
+            if (!is_zkvm) {
+                const api_module = @import("@zeam/api");
```

Comment:
new justification and new finalization is not to be emtted from transition but from chain itself, chain.forkChoice.fcStore has latest_justified and latest_finalized, so emit there using those